docs/components/Address.md

```diff
@@ -31,20 +31,32 @@ There are also custom output properties which can be used to reference component
 - **Theme - Theme** - Configures the styling of component elements
     - **Default:** - See [Theme Documentation](../Theme.md).
 
-The following input properties are records that control whether the respective text input is required or optional, controls the visibility of the respective field and controls the width proportion of the text input. Note, a width proportion of 1 results in the text input width equal to the width of the component.
+The following input properties are records that configure the state of each input field, containing:
+- **Required** - Whether the input is required or optional
+    -**Default:** True
+- **Visible** - Whether the address field is visible or hidden
+    - **Default:** True
+- **InputWidthProportion** - The width proportion of the text input. Note, a width proportion of 1 results in the text input width equal to the width of the component.
+    - **Default:** 1
```

Comment:
this is only the default for some of the fields - could we remove all the defaults from all these descriptions and just have them below?